### ISSUE TITLE
Different entities on day/night

### ISSUE THREAD

**Issue by haavardNO** [danobot/entity-controller#213]
I am not sure if this is a feature request or bug report, since I dont know if EC is suppose to do what I am trying to do.

I have 5 lights in my bedroom.
2 night stand lights
1 led light under bed
1 chandelier 
1 wardrobe light

What I want to do:
Day:
Turn on night stands, led under bed and wardrobe light.

Night:
Turn on only led under bed (rgb value).

Watch all the 5 lights for potential block out etc...

I have tried several approaches, but the below is the current. Any reason for this to not work? When the sensor is triggered it only turns on the listed entities.

```
  motion_light_mb:
    sensor:
      - binary_sensor.ms_motion_master_bedroom
    entities:
      - light.hue_color_candle_1
      - light.hue_color_candle_1_2
      - light.extended_color_light_1
      - light.soverom_tak
    trigger_on_activate: script.day_light_mb
    trigger_on_deactivate: light_off_mb
    delay: 300
    sensor_type: duration
    sensor_resets_timer: True
    block_timeout: 3600
    night_mode:
      delay: 60
      trigger_on_activate: script.night_light_mb
      trigger_on_deactivate: light_off_mb
      start_time: "17:00:00"
      end_time: "06:00:00"
```

**Comment by haavardNO** [comment]
After some more reading I see that different entities on night is not supported.
So I have divided the room in two. And now have no night mode. However, EC is still not running the script, but rather controlling the enteties. Any suggestions?

`  motion_light_room_mb:

    sensor:
      - binary_sensor.ms_motion_master_bedroom
    entities:
      - light.hue_color_candle_1
      - light.hue_color_candle_1_2
      - light.soverom_tak
    trigger_on_activate: script.day_light_mb
    trigger_on_deactivate: script.light_off_mb
    delay: 300
    sensor_type: duration
    sensor_resets_timer: True
    block_timeout: 3600
    overrides:
      - input_boolean.ec_mb_override`

**Comment by haavardNO** [comment]
Ok.
I have tried to narrow it down. A very basic config with a script as result. What am I missing here? The scrips does not fire...

`  
motion_light_room_mb:
    sensor:
      - binary_sensor.ms_motion_master_bedroom
    entity:
      light.hue_color_candle_1

      #entities:
    # - light.hue_color_candle_1
    # - light.hue_color_candle_1_2
    # - light.light_wardrobe_mb
    # - light.soverom_tak
    entity_on: script.day_light_mb
    entity_off: script.light_off_mb
    delay: 300
    sensor_type: duration
    sensor_resets_timer: True
    #block_timeout: 3600
    overrides:
      - input_boolean.ec_mb_override
`

**Comment by danobot** [comment]
You can use ``` to start and end code sections on Github.
You missed a `-` in front of the entity name like you did in `overrides`. Check your logs for warnings

```
entity:
      light.hue_color_candle_1

```

**Comment by haavardNO** [comment]
Thanks for the tips. I tried to use the "insert code" function, but for some reason did not work.
I actually found the fault.
I had a typo in the script name. So I did not call the correct script. Now it works.

However I am seeing a little strange behaviour.
It seams as EC is both controlling the entity with basic "on" command. and firing the script. Is this intended?
My case:
4 lights I want EC to monitor, but the script is set to only turn on 3 lights. 
Result is that all 4 lights are turned on
